packages/muelu/adapters/CMakeLists.txt

```diff
@@ -176,6 +175,13 @@ IF(${PACKAGE_NAME}_ENABLE_EXPLICIT_INSTANTIATION)
     )
   ENDIF()
 
+  IF (${PACKAGE_NAME}_ENABLE_Belos AND ${PACKAGE_NAME}_ENABLE_Ifpack2 AND ${PACKAGE_NAME}_ENABLE_Experimental AND HAVE_TEUCHOS_COMPLEX)
+    APPEND_GLOBAL_SET(MUELUADAPTERS_ETI_CLASSES
+      MueLu::ShiftedLaplacianOperator
+      MueLu::ShiftedLaplacian
+    )
+  ENDIF()
```

Comment:
Should this be modified instead in `packages/muelu/src/Utils/ClassList/SC-LO-GO-NO.classList`?